include/swift/AST/Decl.h

```diff
@@ -5610,6 +5610,9 @@ class FuncDecl : public AbstractFunctionDecl {
   bool isStatic() const {
     return Bits.FuncDecl.IsStatic;
   }
+  bool isCallable() const {
+    return getName().str() == "call" && isInstanceMember();
```

Comment:
Consider adding a `IsCallable` bit to `FuncDecl`, instead of checking "callable-ness" on the fly.
That requires further decl/serialization changes - we can make that change after committing our desired `func call` API.

Reply:
I'd recommend *not* doing that, unless this is a particularly hot predicate.  Cached bits like this have to be kept in sync with the ast.  Also, instead of serializing it, if the bit needs to be stored, it can always be computed in the module reader.